csm/core/services/users.py

```diff
@@ -213,6 +213,12 @@ async def get_user_list(self, limit, offset, sort_by, sort_dir):
 
         return [self._user_to_dict(x) for x in user_list]
 
+    async def get_user_count(self):
+        """
+        returns the count of existing users
```

Comment:
Minor: start the comment with a capital letter. PEP also suggests 'Return' instead of 'Returns'.